code/parse/sexp.h

```diff
@@ -925,6 +943,11 @@ const char *CTEXT(int n);
 // flags for sexpressions -- masked onto the end of the type field
 #define SEXP_FLAG_PERSISTENT				(1<<31)		// should this sexp node be persistant across missions
 #define SEXP_FLAG_VARIABLE					(1<<30)
+#define SEXP_FLAG_LIST_CONTAINER			(1<<29)
```

Comment:
Not sure that separate flags for list container vs. map container are needed. What are these sorts of flags for, anyway?

Reply:
I seem to remember that the code used them to actually tell the difference between the two types. I can't remember if I later changed that.